ocs_ci/ocs/perfresult.py

```diff
@@ -105,8 +104,9 @@ def es_write(self):
                 index=self.new_index, doc_type="_doc", body=self.results, id=self.uuid
             )
         except ESExp.RequestError as e:
-            log.error(f"Failed writhing data with {e}")
-            raise
+            log.warning(f"Failed writhing data with {e}")
```

Comment:
Spelling fix: writing instead of writhing

Reply:
fixed it